main.rb

```diff
@@ -1,12 +1,36 @@
 require_relative 'app'
+require 'json'
 
-# Main
-def main
-  app = App.new
-  loop do
-    app.run
+class Main
+  def initialize
+    @app = App.new
+  end
+
+  def actions
     puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
```

Comment:
- [ ] Kindly ensure the rental details are listed as well.

  ![duee](https://user-images.githubusercontent.com/81468675/211836747-ab255536-cdb6-42ad-b7f0-b87e91f6577f.jpg)